spec/forms/state_file/nj_disabled_exemption_form_spec.rb

```diff
@@ -0,0 +1,94 @@
+require "rails_helper"
+
+RSpec.describe StateFile::NjDisabledExemptionForm do
+  describe "validations" do
```

Comment:
NIT: There's no need to rewrite these, but in the future you may consider leveraging [shoulda-matchers](https://github.com/thoughtbot/shoulda-matchers). An example of setting them up is in the `nc_state_id` [shared example group](https://github.com/codeforamerica/vita-min/blob/main/spec/support/shared_examples/nc_state_id.rb) (but it doesn't have to be a shared example group ofc). These are most suitable when the validation is not terribly custom, which doesn't seem to be the case here.

Reply:
Appreciate the resource! Was not familiar with shoulda-matchers before this!